docs/configuration.md

```diff
@@ -1564,8 +1564,8 @@ spark.sql("SET -v").show(n=200, truncate=False)
 <div data-lang="r"  markdown="1">
 
 {% highlight r %}
-# sqlContext is an existing sqlContext.
-properties <- sql(sqlContext, "SET -v")
+# spark is an existing SparkSession.
+properties <- sql(spark, "SET -v")
```

Comment:
I believe in the most recent SparkR API, you don't need to pass in the `spark` parameter to run a SQL statement now.